app/locales/fr-fr.json

```diff
@@ -174,5 +174,7 @@
   "Close": "Fermé",
   "Contribute now": "Participer",
   "Help us uncover and predict the corona spread in your area": "Aidez-nous à <span>estimer</span> et <span>prédire</span> la propagation du corona virus dans votre région",
-  "Numbers": "Chiffres"
+  "Numbers": "Chiffres",
+  "In isolation due to orders from the government": "In isolation due to orders from the government",
+  "In isolation due to COVID-19 infection": "In isolation due to COVID-19 infection"
```

Comment:
"In isolation due to orders from the government": "isolé(e) en raison des ordres du gouvernement",
  "In isolation due to COVID-19 infection": "isolé(e) car vous êtes infecté(e) par le COVID-19"

Reply:
@fossecode 
Is there something else?